ui/Screen/Onboarding/EnterPassphrase.svelte

```diff
@@ -61,14 +67,16 @@
       constraints
     );
 
+    // @ts-expect-error: `validations` is guaranteed to be a validations object
     passphraseValidation = getValidationState("passphrase", validations);
     repeatedPassphraseValidation = getValidationState(
       "repeatedPassphrase",
+      // @ts-expect-error: `validations` is guaranteed to be a validations object
       validations
     );
   };
 
-  $: validate(passphrase, repeatedPassphrase);
+  $: validate();
```

Comment:
Is this a typo?

Reply:
Because if you don't list the variables the reactive statement won't trigger.

Reply:
`validate` does not have any arguments. The original code was an oversight.

Reply:
I guess we're just abusing the parameters. Not sure it'll work, but maybe we can rewrite it to something like this, to make it more explicit?
```
$: passphrase; repeatedPassphrase; validate();
```

Reply:
3bedfee09431dd6318216b48882c2c8a2d681122 -> this also works 👍